ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -120,8 +120,8 @@ type structure_interface = {
     "netmask" ? type_ip
     "broadcast" ? type_ip
     "driver" ? string
-    "bootproto" ? string
-    "onboot" ? string
+    "bootproto" ? string # TODO: only allow match(SELF, '^(static|bootp|dhcp)')
+    "onboot" ? string # TODO: only allow match(SELF, '^(yes|no)$')
```

Comment:
Or use a real boolean?

Reply:
ok, i'll chnage this in a separte commit (it just introduces more backwards incompatibility)

Reply:
The TODO could be to use a real boolean (or the transitional type) but I don't like leaving too many TODOs lying around